src/main/java/mat/server/service/FhirValidationReportService.java

```diff
@@ -98,22 +97,17 @@ private String generateValidationReport(ConversionResultDto conversionResultDto,
         if (conversionResultDto != null && measure != null) {
             //Measure FHIR validation errors
             List<FhirValidationResult> measureFhirValidationErrors = conversionResultDto.
-                    getMeasureConversionResults().
-                    getMeasureFhirValidationErrors();
-
-            //Library FHIR validation errors
-            LibraryConversionResults libraryConversionResults = conversionResultDto.getLibraryConversionResults();
-            List<FhirValidationResult> libraryFhirValidationErrors = libraryConversionResults.getLibraryFhirValidationErrors();
-
-            //CQL conversion errors
-            List<CqlConversionError> cqlConversionErrors = libraryConversionResults.
-                    getCqlConversionResult().
-                    getCqlConversionErrors();
-
-            //ValueSet FHIR validation errors
-            List<ValueSetValidationResult> valueSetFhirValidationErrors = conversionResultDto.
-                    getValueSetConversionResults().
-                    getValueSetFhirValidationErrors();
+                    getMeasureConversionResults().getMeasureFhirValidationResults();
+
+            // Library FHIR validation errors
+            List<LibraryConversionResults> libraryConversionResults = conversionResultDto.getLibraryConversionResults();
```

Comment:
It's a temporary fix to make the code compilable and do integration testing. The logic should be adjusted during real validation service implementation.